index.js

```diff
@@ -89,11 +118,11 @@ Your function should accept:
 
 For example, getFlavorByIndex(originalFlavors, 2) would return "Black Walnut", assuming Rainbow Sherbert has been added successfully. */
 
-function getFlavorByIndex(/*code here*/){
-
+function getFlavorByIndex(string, arr){originalFlavors
```

Comment:
Since you're targeting the index with bracket notation, you can target it like so:
```
function getFlavorByIndex(arr, index) {
  return arr[index]
}
console.log(getFlavorByIndex(originalFlavors, 2))
```